client/app/scripts/superdesk-packaging/packaging.js

```diff
@@ -467,7 +467,7 @@
             link: function(scope) {
                 scope.data = null;
                 scope.error = null;
-                scope.type = scope.item.type || scope.item.itemClass.split(':')[1];
```

Comment:
they have it, but there is no need to use it because it's fetching the item anyhow.
so you can move this `scope.type = ` part after `scope.data = result;` or you can change template to use `data.type`